action.yml

```diff
@@ -40,14 +40,22 @@ inputs:
     type: boolean
     description: 'If true, the action will not validate the user or the commit verification status'
     default: false
+  event-name:
+    type: string
+    description:
+    default: pull_request
```

Comment:
Seems like the issue of missing quotes, it should be `default: 'pull_request'`

Reply:
@climba03003 @simoneb Can someone re-run that in debug mode (I don't have permissions) - I don't quite think this is a quoting issue, but I'd like to see how the evaluation of `${{ inputs.event-name }}` goes.

Reply:
@diranged I tried a few combinations of with-and-without quotes without success, may I ask some help to fix this? The easiest way to go about it is that you create a branch for the action, where you try your fixes, and push it to a branch on your own remote. 

Then you set up a repo to try out the fix where you reference your own version of the action, meaning that assuming that you call the branch "fix-event-name", you'd reference it in the workflow as

```yaml
uses: diranged/github-action-merge-dependabot@fix-event-name
```